CONTRIBUTING.adoc

```diff
@@ -87,6 +87,10 @@ In order to do builds locally, install the following build dependencies:
 ----
 $ sudo dnf install golang golang-race make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel
 ----
+The following might also me useful if you prefer as much of the build and test chain to be installed via rpm packages:
```

Comment:
Two typo fixes: s/me/be/ and s/as much of the build and test chain/& as possible/